azure-functions-gradle-plugin/src/main/java/com/microsoft/azure/plugin/functions/gradle/handler/DeployHandler.java

```diff
@@ -166,6 +154,19 @@ public void execute() throws AzureExecutionException {
         listHTTPTriggerUrls(app);
     }
 
+    private RuntimeConfig getRuntimeConfig() {
+        final GradleRuntimeConfig runtime = ctx.getRuntime();
```

Comment:
Do we still need `GradleRuntimeConfig` can we get config for task from ctx directly?

Reply:
all the properties in GradleRuntimeConfig is string

```java
public class GradleRuntimeConfig {
    private String os;
    private String javaVersion;
    private String webContainer;
    private String image;
    private String registryUrl;
    private String username;
    private String password;
    private String startUpCommand;
}

```